dns-records.tf

```diff
@@ -12,6 +12,19 @@ resource "azurerm_dns_a_record" "cert-ci-jenkins-io" {
   tags = local.default_tags
 }
 
+# A record for the jenkinsistheway.io redirector hosted on publick8s redirecting to stories.jenkins.io
+resource "azurerm_dns_a_record" "jenkinsistheway_io" {
+  name                = "@"
+  zone_name           = azurerm_dns_zone.jenkinsistheway_io.name
+  resource_group_name = azurerm_resource_group.proddns_jenkinsisthewayio.name
+  ttl                 = 60
+  records              = ["20.119.232.75", "2603:1030:408:7::44"] # publick8s_public_ipv4_address & publick8s_public_ipv6_address, defined in https://github.com/jenkins-infra/azure/blob/main/publick8s.tf
```

Comment:
Post merge info:

a DNS `A`record only supports IPv4. This should be changed.

(A https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/dns_aaaa_record resource is needed for the IPv6)

Reply:
🤦